### DISCUSSION TITLE
How to get info dissociated of ECU-C from imported and producted energy

### DISCUSSION THREAD

**Discussion by Alejandro67120** [skelgaard/homeassistant-apsystems#27] [Q&A]
Hello, 
I would like to know by witch way i could get the info dissociated from imported and producted energy from my ECU C ?
The sensor apsystems_energy_day show the totality.

And i would like to dissociate these information to use them into the energy panel, but i'm kinda a newby of these kind of configuration 

Below, a screen of the data i could have on APS website
![image](https://user-images.githubusercontent.com/101482157/233682668-32619ffc-39db-428f-a9b7-1175b8649129.png)

Thanks for help :)

**Comment by skelgaard** [comment]
as i told in https://github.com/skelgaard/homeassistant-apsystems/issues/26 there is no official way to get this info... you can look in what requests your browser does and use them.
I only have the info that is in the call i make and cannot give you anything more.

**Reply by Alejandro67120** [reply]
I guess there is a possibility to view them from the ECU because there is a TOR on data from the GRID and another TOR on data from panel.

You mean you cannot isolated them because you have not these kind of configuration ? 
I would be happy to help if i can to improve these plugin for everyone :)

**Reply by skelgaard** [reply]
i mean there is no documentation....
Go to the apsystem homepage, look in your browsers request if there is one that comes whith what you need and then test it out if you can make it work in postman.
If you can that, then a module for it can be made or modifies this to include that info.

**Reply by gregoo31** [reply]
I bought the apsystem application for jeedom, the energy consumed is well indicated (I no longer use jeedom because many paid plugins are not followed by the developers... in particular that of the rflink for my somfy roller shutters) I start quietly with Home assistant ;-)

Can we retrieve the API request information in the jeedom apsystem application?
I have a very very low level in programming!
